index.rst

```diff
@@ -72,30 +72,31 @@ Prerequisites
     *  **Clean** environment, only test-related images, containers
        and dependent services (besides ``docker -d``) at the start
        of **every** Autotest run.
-    *  Docker installation (with ``docker -d`` running at startup)
+    *  Docker installation (with docker daemon running)
     *  Default settings for Docker on your platform/OS,
-       unless otherwise noted.
+       including storage (LVM thin-pool + XFS).
     *  Sufficient available temporary storage for multiple copies of the
        test-related images/container content.
 
 *  `Supported Docker OS platform`_
 
     *  Red Hat Enterprise Linux 7 Server
-    *  Fedora 20 or later recommended
-    *  Linux kernel 3.12.9 or later
+    *  Red Hat Enterprise Linux Atomic Host
+    *  Fedora 22 or later including Atomic
```

Comment:
Last time I tried (February?), docker-autotest just didn't work on Fedora. Has that changed?